pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -266,7 +267,7 @@ func NewContainerManager(mountUtil mount.Interface, cadvisorInterface cadvisor.I
 		cm.Lock()
 		defer cm.Unlock()
 		for k, v := range updates {
-			if v.Value() <= 0 {
+			if v.Value() < 0 {
```

Comment:
When is the value less than 0 ?

Reply:
The comment above handler.garbageCollectionFunc definition explains how this happens. I also added some comment here to explain the scenario.